aderyn_core/src/detect/detector.rs

```diff
@@ -37,6 +37,8 @@ use std::{
     str::FromStr,
 };
 
+use super::high::UncheckedReturnDetector;
```

Comment:
This may be your IDE auto-importing here. We should add these `use` statements to the large ones above ^. I've fixed in my dev merge commit :)